<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>860</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
border-image: url(:/imagenes/background-1789175_960_720.png);
}</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>860</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="salir">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#salir{
background-color: rgba(0,0,0,0);
color:white;
border-top-right-radius: 25px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#salir:hover{
background-color: rgba(0,0,0,40);
color:white;
border-top-right-radius: 25px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#salir:pressed{
background-color: rgba(0,0,0,80);
color:white;
border-top-right-radius: 25px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="min">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#min{
background-color: rgba(0,0,0,0);
color:white;
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#min:hover{
background-color: rgba(0,0,0,40);
color:white;
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#min:pressed{
background-color: rgba(0,0,0,80);
color:white;
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_yo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_computadora">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>150</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Britannic Bold&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Yo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Britannic Bold&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Computadora&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="piedra">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>280</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#piedra{
background-color: rgb(170, 170, 255);
font: 20pt &quot;Britannic Bold&quot;;
border: 2px solid white;
border-radius: 25px;
}

QPushButton#piedra:hover{
background-color: rgb(130, 130, 195);
}

QPushButton#piedra:pressed{
background-color: rgb(95, 95, 143);

}</string>
     </property>
     <property name="text">
      <string>Piedra</string>
     </property>
    </widget>
    <widget class="QPushButton" name="papel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>340</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#papel{
background-color: rgb(170, 170, 255);
font: 20pt &quot;Britannic Bold&quot;;
border: 2px solid white;
border-radius: 25px;
}

QPushButton#papel:hover{
background-color: rgb(130, 130, 195);
}

QPushButton#papel:pressed{
background-color: rgb(95, 95, 143);

}</string>
     </property>
     <property name="text">
      <string>Papel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tijera">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>400</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#tijera{
background-color: rgb(170, 170, 255);
font: 20pt &quot;Britannic Bold&quot;;
border: 2px solid white;
border-radius: 25px;
}

QPushButton#tijera:hover{
background-color: rgb(130, 130, 195);
}

QPushButton#tijera:pressed{
background-color: rgb(95, 95, 143);

}</string>
     </property>
     <property name="text">
      <string>Tijera</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_msg">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>180</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
